src-tauri/src/lib.rs

```diff
@@ -107,6 +107,24 @@ pub fn find_game_install_location() -> Result<GameInstall, String> {
         }
     };
 
+    // Other custom locations (for dev)
+    #[cfg(debug_assertions)]
+    {
+        let locations = [
```

Comment:
By experience, even if you leave a note saying "DON'T COMMIT THIS EVER", somebody will end up doing it accidentally anyways :)
The proper way of doing this would be to load the debug location path from an environment file (`.env`) that is added to `.gitignore`, to avoid being committed at all.